src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,64 @@
+package lotto.controller;
+
+import lotto.domain.LottoResultStatistics;
+import lotto.domain.lottos.LottoTickets;
+import lotto.domain.lottos.winnerlotto.LottoWinner;
+import lotto.domain.lottos.winnerlotto.LottoWinnerBonusNumber;
+import lotto.domain.lottos.winnerlotto.LottoWinnerTicket;
+import lotto.domain.money.Money;
+import lotto.service.LottoTicketService;
+import lotto.service.LottoTicketsService;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+public class LottoController {
+
+    public void lottoStart() {
+        Money money = initMoney();
+        LottoTickets lottoTickets = initLottoTickets(money);
+
+        LottoWinnerTicket lottoWinnerTicket = initLottoWinnerTicket();
+        LottoWinnerBonusNumber lottoWinnerBonusNumber = initLottoWinnerBonusNumber(lottoWinnerTicket);
+        LottoWinner lottoWinner = new LottoWinner(lottoWinnerTicket, lottoWinnerBonusNumber);
+
+        LottoResultStatistics lottoResultStatistics =
+                LottoResultStatistics.calculateResultStatistics(lottoTickets, lottoWinner);
+        OutputView.printRewardResultBoard(lottoResultStatistics);
+        OutputView.printFinalResult(lottoResultStatistics, money);
+    }
+
+    private Money initMoney() {
+        Money money;
+        try {
+            money = new Money(InputView.getMoneyInput());
+        } catch (NullPointerException | IllegalArgumentException e) {
+            OutputView.printErrorMessage(e);
+            return initMoney();
+        }
+        return money;
+    }
```

Comment:
아래처럼 구현해도 될 것 같아요 :) 
```suggestion
    private Money initMoney() {
        try {
            return new Money(InputView.getMoneyInput());
        } catch (NullPointerException | IllegalArgumentException e) {
            OutputView.printErrorMessage(e);
            return initMoney();
        }
    }
```

Reply:
굉장히 부끄러운 부분이네요 😳😳😳

01f96c2